pkg/commands/backend/notes/create.txt

```diff
@@ -0,0 +1,2 @@
+API endpoint information:
```

Comment:
In future we would flesh this out to have more specific information relevant to the flags implemented in the CLI command (e.g. we would say something along the lines of "read the following API documentation to understand the model for the --example flag"), but for now this will suffice as an example of injecting custom information.